ci-operator/step-registry/ipi/conf/ipi-conf-commands.sh

```diff
@@ -49,3 +49,10 @@ if [ -n "${PUBLISH}" ]; then
 publish: ${PUBLISH}
 EOF
 fi
+
+if [ -n "${CREDENTIALS_MODE}" ]; then 
+        echo "Adding 'credentialsMode: ...' to install-config.yaml"
+        cat >> "${out}" << EOF
```

Comment:
I remember wking has ever mentioned we should use patch command to replace cat append operations, so suggest to use patch command here, is possible, also suggest to modify the above publish setting command lines.